app/src/test/java/com/ably/tracking/demo/publisher/ably/FakeAssetTracker.kt

```diff
@@ -17,7 +16,7 @@ class FakeAssetTracker : AssetTracker {
 
     val trackables = MutableStateFlow(emptySet<Trackable>())
 
-    override fun connect(clientId: String): SharedFlow<Set<Trackable>> {
+    override suspend fun connect(clientId: String): SharedFlow<Set<Trackable>> {
```

Comment:
Like above, why does it need to be `suspend`? :thinking:

Reply:
https://github.com/ably/asset-tracking-android-rider-app-demo/pull/34/commits/87c991b122230f891131b877c04bc3622fc99e08